app/tests/test_basic.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+"""Tests for Basic Functions"""
+import sys
+import json
+import unittest
+
+sys.path.append("../..")
+from app.main import *
+
+
+class TestFunctions(unittest.TestCase):
+
+    def setup(self):
+        app.app.config['TESTING'] = True
+        self.app = app.app.test_client()
+
+    def test_output(self):
+        with app.test_request_context():
+            out = output('error','Test Error','local_host')
+            response = [
```

Comment:
What if we have all the responses in seperate file. and we just call them

Reply:
I think it will increase the test execution time and I also have taken reference from open-event server unit tests. I found it most suitable. 👍

Reply:
It will take time to write all tests, I will send multiple PRs for this issue. Please merge this and give me time as I am new to it.

Reply:
@ParthS007 Please explain: if we have all the responses in a separate file, it's better to call from there. How will it increase the test of execution?